### PR TITLE
Restore backward compatibility by adding back client.parse.env() method

### PR DESCRIPTION
**Pull request by asvetlov**

### CODE REVIEW

File: neuromation/api/parser.py [hunk 1]
```diff
@@ -144,6 +145,16 @@ def _remote_to_local_image(self, image: RemoteImage) -> LocalImage:
         )
         return parser.convert_to_local_image(image)
 
+    def env(self, env: Sequence[str]) -> Tuple[Dict[str, str], Dict[str, URL]]:
```
Comment (serhiy-storchaka) [inline neuromation/api/parser.py:148]: Should not it have the `env_file` parameter?

**Review by serhiy-storchaka — APPROVED** [review]

**Comment by serhiy-storchaka** [comment]
`neuro-flow` alos expects that `client.parse.volumes()` returns a 2-tuple. Maybe add `__iter__` to `VolumeParseResult` which returns `iter((self.volumes, self.secret_files))`?

File: neuromation/api/parser.py (hunk 1 above)
Comment (asvetlov) [reply neuromation/api/parser.py]: Good catch. Added

**Comment by asvetlov** [comment]
> neuro-flow alos expects that client.parse.volumes() returns a 2-tuple. Maybe add __iter__ to VolumeParseResult which returns iter((self.volumes, self.secret_files))

Thanks!
I've implemented `__len__()` / `__getitem__()` pair.

**Review by serhiy-storchaka — APPROVED** [review]

**Pull request merged**